### DISCUSSION TITLE
Problem with the column in the site Joomla

### DISCUSSION THREAD

**Discussion by SalvoLaDelfa** [joomla/joomla-cms#39555] [General]
Hi. If I put two column in the home of my site  the homepage shrinkes (the articles go to one column.
Could you help me? Thank you
The version of Joomla is 3.11.12
[Site Problem.pdf](https://github.com/joomla/joomla-cms/files/10342191/Site.Problem.pdf)

**Comment by ghost** [comment]


**Reply by brianteeman** [reply]
@joomlaphp the user is using j3 not j4

**Reply by ghost** [reply]
You're right, sorry.

**Comment by chmst** [comment]
Probably your template uses an override which does not work with columns

**Comment by brianteeman** [comment]
The template does support two columns as they are using it successfully on many many pages https://www.ilfilodies.it/index.php/psicologia

The problem is that you have a div missing somewhere in an article (perhaps the first one) and as a result all articles are being nested inside of each other. which is why you get the errors you are seeing.

I suggest you unpublish all the articles on the frontpage. set it to 2 columns. Then publish one article and check you have two columns and keep publishing articles until it breaks. Then you know which article has the problem.

**Comment by SalvoLaDelfa** [comment]
Is it possibile to understand where is  the div missing? Is there a checker to find it?

Thank you

**Reply by SalvoLaDelfa** [reply]
Yes, Some program tha can help me to solve the problem as soon as possible
